cpp/rmm_log.txt

```diff
@@ -0,0 +1 @@
+[ 23294][23:00:25:246785][info  ] ----- RMM LOG BEGIN [PTDS DISABLED] -----
```

Comment:
What is this file?

Reply:
Generated by RMM, but only in the current directory of execution... We should add it to .gitignore